### PR TITLE
GUI/map/entities

### PR DESCRIPTION
**Pull request by GuyomT**
Adding the parsing of the response of the server and display

### CODE REVIEW

File: src/zappy_gui_src/Core/SFML/Map/Map.hpp [hunk 1]
```diff
@@ -130,6 +135,8 @@ namespace gui {
 
             /// \brief Display players that are in the tile, if there is at least one player.
             void _displayPlayers(Tile &tile, sf::CircleShape &playerRepresentation);
+            void _displayResources(Tile &tile, sf::CircleShape &resourcesRpresentation);
+            void _displayEggs(Tile &tile, sf::CircleShape &eggRpresentation);
```
Comment (osvegn) [inline src/zappy_gui_src/Core/SFML/Map/Map.hpp:139]: missing documentation

**Review by osvegn — APPROVED** [review]
Loooooook goooooood toooooo meeeeee

**Review by pariciadra — COMMENTED** [review]
Split is the keywork of the review

**Review by pariciadra — CHANGES_REQUESTED** [review]
Split is the keywork of the review

**Review by LucasTesnier — APPROVED** [review]
NTR!

**Pull request merged**